modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -344,6 +346,9 @@ bool QueueReader::getInputVector(std::vector<Q*> &in_queues,
                                  cv::GRunArgs    &in_constants,
                                  cv::GRunArgs    &isl_inputs)
 {
+    GAPI_ITT_STATIC_LOCAL_HANDLE(inputs_hndl, "inputs_awaiting_region");
```

Comment:
I'd drop `_region` from everywhere, they all are regions or tasks. Longer names harder to review in the traces

Reply:
Btw let's call it `getInputVector`

Reply:
Renamed, thanks!